src/App.tsx

```diff
@@ -1,9 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const queryChange = (value: string) => {
+    setQuery(value);
+  };
```

Comment:
Take event as a prop, so you can pass handler easier, also you can covert it to `lowerCase` here
```suggestion
  const queryChange = (event) => {
    setQuery(event.target.value.toLowerCase());
  };
```